compiler/codegen.js

```diff
@@ -5423,6 +5425,29 @@ const internalConstrs = {
     type: TYPES.undefined,
     notConstr: true,
     length: 1
+  },
+
+  __Porffor_rawType: {
+    generate: (scope, decl) => {
+      const type = getNodeType(scope, decl.arguments[0]);
+      if (type) {
```

Comment:
`getNodeType` never returns a falsy value?

Reply:
I think if `getNodeType` didn't assume types then I think this might be required, but that being said it does :P
I've now removed the check and the builtin.